podcasts/PodcastManager.swift

```diff
@@ -91,6 +91,14 @@ class PodcastManager: NSObject {
             }
         }
 
+        // check if any existing episodes were downloading, but aren't currently (caused by app force quit while episode was downloading)
+        let stuckDownloadingEpisodes = DataManager.sharedManager.findEpisodesWhere(customWhere: "episodeStatus == ?", arguments: [DownloadStatus.downloading.rawValue])
```

Comment:
@jgcaruso if someone configured the app to auto-download only on Wi-fi, I believe this will bypass this setting.

The way it works now is if someone closes the app is that downloads won't be resumed, so it's up to the user to add them again. With this change, downloads will resume as soon as the user opens the app. If they have 10 episodes to download, this will download from their cellular data.

While your change works as expected, I wonder whether not resuming pending downloads was an intentional decision. I think we can check how that works on Android. :)

Reply:
> With this change, downloads will resume as soon as the user opens the app — it doesn't matter if on Wi-Fi or not.

Hmm good point. I suppose an extra check to the Podcast's setting for "only download on wifi" could be done before re-starting the download, and fixing the status to `waitinForWifi` if the device is currently on cellular, so it gets picked up next time the device is on wifi.

Reply:
@jgcaruso that looks like a good solution!